src/Autosuggest.js

```diff
@@ -50,6 +50,7 @@ class Autosuggest extends Component {
     id: PropTypes.string.isRequired,
     inputRef: PropTypes.func.isRequired,
     shouldHideSuggestions: PropTypes.func.isRequired,
+    getDescriptionId: PropTypes.func,
```

Comment:
Please set a default value for this prop

Reply:
Good catch, thanks! This one is actually required, though. I'm setting the default value in the container.